dialogflow/stopwatch/Stopwatch/StopwatchService.swift

```diff
@@ -15,14 +15,12 @@
 //
 import Foundation
 import googleapis
-import AuthLibrary
 
 let Host = "dialogflow.googleapis.com"
-let ProjectName = "your-project-identifier" // UPDATE THIS
+// TODO: Replace with your GCP PROJECT_ID
+let ProjectName = "your-project-identifier"
```

Comment:
nit: let ProjectId = "YOUR_PROJECT_ID"
We typically use project id on our samples, but no need to fix. :)

Side question: Android has like a `strings` file to put app level strings that might need to be used. Does iOS have such a thing that this can be moved into to make it easier to setup the app?

Reply:
Yeah, We have defined constants struct to handle all types of string literals, I will move all the string constants in Constants struct